docs/cylc-8-tasks.md

```diff
@@ -118,7 +118,7 @@ o-------------------------------o-----------------------------------------------
     - awaiting decision to go ahead with this
 
 - Chat Platform
-  - ~~Slack?~~ - free account is lossy
+  - ~~Slack?~~ - free account is lousy
```

Comment:
Haha!  Maybe true, but I did mean "lossy" - in that you lose access to your information after 10k posts (or something like that!)

Reply:
Ah, I see! I don't think I have ever seen 'lossy' as an adjective in that way, hence why I thought it was a typo. Lossy does make sense in context. I'll get this line reverted.